<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cg3::viewer::DrawableMeshDrawListManager</class>
 <widget class="QFrame" name="cg3::viewer::DrawableMeshDrawListManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QRadioButton" name="tColorRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/vcolor.png</normaloff>:/vcolor.png</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="smoothRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/smooth.png</normaloff>:/smooth.png</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="vColorRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/tcolor.png</normaloff>:/tcolor.png</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QLabel" name="nFacesLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="wireframeCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/wire.png</normaloff>:/wire.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="savePushButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/save.png</normaloff>:/save.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="pointsRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/points.png</normaloff>:/points.png</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="flatRadioButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/flat.png</normaloff>:/flat.png</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSlider" name="wireframeWidthSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="nVerticesLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QCheckBox" name="subFrameAxisCheckBox">
     <property name="text">
      <string>Draw SubFrame Axis</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="bboxCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/bbox.png</normaloff>:/bbox.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Wireframe Width:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Point Width:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSlider" name="pointWidthSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
